CodeEdit/Features/WindowCommands/FileCommands.swift

```diff
@@ -79,6 +83,16 @@ struct FileCommands: Commands {
 
             Divider()
 
+            Button("Workspace Settings") {
+                NSApp.sendAction(#selector(CodeEditWindowController.openWorkspaceSettings(_:)), to: nil, from: nil)
+            }
+            .disabled(windowController?.workspace == nil)
+            .onReceive(NSApp.publisher(for: \.keyWindow)) { window in
+                windowController = window?.windowController as? CodeEditWindowController
+            }
```

Comment:
If a user spams this button, the window will open multiple times, i.e. many windows are stacked on top of each other.